WalletWasabi.Gui/Global.cs

```diff
@@ -513,15 +511,14 @@ private void Coins_CollectionChanged(object sender, NotifyCollectionChangedEvent
 							continue;
 						}
 						string amountString = coin.Amount.ToString(false, true);
-						using (var process = Process.Start(new ProcessStartInfo
+						using var process = Process.Start(new ProcessStartInfo
```

Comment:
Simple `Using` statements are less efficient and I still think they are less readable.